ouroboros-consensus-diffusion/src/ouroboros-consensus-diffusion/Ouroboros/Consensus/Node/GSM.hs

```diff
@@ -78,11 +81,23 @@ data CandidateVersusSelection =
     -- ^ Whether the candidate is better than the selection
   deriving (Eq, Show)
 
+data GsmState =
```

Comment:
Is it worth clarifying what GSM is in the comments?

Reply:
Added a comment expanding the acronym (it is also in the module header, but that is a bit distanced)